base.mako

```diff
@@ -59,6 +59,7 @@
         <footer class="pc-footer"><div class="container"></div></footer>
         <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
         <script src="node_modules/bootstrap-sass/assets/javascripts/bootstrap.min.js"></script>
+        <script>!function(){var e=document.getElementsByClassName("highlight");function t(e){var t=document.createElement("textarea");t.value=e,t.style.top="0",t.style.left="0",t.style.position="fixed",document.body.appendChild(t),t.focus(),t.select(),document.execCommand("copy"),document.body.removeChild(t)}for(let o of e){var n=document.createElement("img");n.setAttribute("src","./copy-icon.svg"),n.setAttribute("alt","copy"),n.setAttribute("title","Copy code"),n.setAttribute("width","16"),n.setAttribute("height","16"),o.insertBefore(n,o.children[0]),n.addEventListener("click",function(){t(o.getElementsByTagName("pre")[0].innerText)})}}();</script>
```

Comment:
non-essential scripts should go at the end of body, also I'd rather not minify this in the source (it makes it much harder to modify)

Reply:
done => 02ae85e